Description: Install the translations in $FILESDIR/lang instead of /usr/bin
Author: Joachim Reichel <reichel@debian.org>

Index: cppcheck/gui/CMakeLists.txt
===================================================================
--- cppcheck.orig/gui/CMakeLists.txt
+++ cppcheck/gui/CMakeLists.txt
@@ -51,7 +51,7 @@ if (BUILD_GUI)
     endif()
 
     install(TARGETS cppcheck-gui RUNTIME DESTINATION ${CMAKE_INSTALL_FULL_BINDIR} COMPONENT applications)
-    install(FILES ${qms} DESTINATION ${CMAKE_INSTALL_FULL_BINDIR} COMPONENT applications)
+    install(FILES ${qms} DESTINATION ${FILESDIR}/lang COMPONENT applications)
 
     install(FILES cppcheck-gui.desktop DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/applications)
 
